src/sentry/static/sentry/app/utils/withTeamsForUser.tsx

```diff
@@ -52,13 +53,27 @@ const withTeamsForUser = <P extends InjectedTeamsProps>(
         loadingTeams: true,
       });
       try {
+        metric.mark('user-teams-fetch-start');
         const teamsWithProjects: TeamWithProjects[] = await this.props.api.requestPromise(
           this.getUsersTeamsEndpoint()
         );
-        this.setState({
-          teams: teamsWithProjects,
-          loadingTeams: false,
-        });
+        this.setState(
+          {
+            teams: teamsWithProjects,
+            loadingTeams: false,
+          },
+          () => {
+            metric.measure({
+              name: 'app.component.perf',
+              start: 'organization-details-fetch-start',
```

Comment:
Should this match the mark from before the request started?

Reply:
oh my that's embarassing